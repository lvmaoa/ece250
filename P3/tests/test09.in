load
size
e carmen
e carmen
e carmen
size
empty
s carmen
spellcheck carmen
i carmen
spellcheck carmen
size
e carmen
size
e an
e and
exit

