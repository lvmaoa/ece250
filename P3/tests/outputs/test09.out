success
number of words is: 274
success
failure
failure
number of words is: 273
empty 0
not found
card 
success
correct
number of words is: 274
success
number of words is: 273
success 
success
